getyour/manage.py

```diff
@@ -6,7 +6,7 @@
 
 def main():
     """Run administrative tasks."""
-    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'getyour.settings.dev')
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'getyour.settings.local')
```

Comment:
Our test Azure app instance relies on the default of `getyour.settings.dev`.

Reply:
Same issue here with my local dev setup